src/components/MovieDetails.tsx

```diff
@@ -1,9 +1,108 @@
-const MovieDetails = () => {
-  return (
-    <div>
-      <p>Movie details</p>
+import { useState, useEffect } from 'react';
+
+import api from 'src/service/api';
+
+import ReadMore from './ReadMore';
+
+import '../styles/MovieDetails.scss';
+
+type MovieDetailsProps = {
+  match: { params: { id: string } }
+}
+
+const MovieDetails = ({ match }: MovieDetailsProps) => {
+
+  const [movieId, setMovieId] = useState<string>(' ');
```

Comment:
Is space added by purpose?

Reply:
i guess yes, but implementation was changed and works with both ways) 
deleted empty space.